scripts/deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+kubectl label nodes $NODE_SELECTOR nym=true
```

Comment:
As soon as the pod is redeployed it can be scheduled on a different node. The idea of using the LoadBalancer service is having a stable IP to reach the service from outside, you should get the load balancer IP here instead

Reply:
https://github.com/w3f/nym-mixnode-chart/blob/init/chart/mixnode/values.yaml#L57-L58
this will bring it to the same node.

Reply:
If the node is recycled then the label is lost. One of the big advantages of kubernetes is forgetting about where the workloads actually run, I think we should not try to work around this.
We have currently a service of type LoadBalancer, why not using it?